### PR TITLE
changes from live gatherer box

### PR DESCRIPTION
**Pull request by amj**

### CODE REVIEW

File: example_buffer.py [hunk 1]
```diff
@@ -68,20 +68,18 @@ def parallel_fill(self, games, threads=8):
         """ games is a list of .tfrecord.zz game records. """
         games.sort(key=os.path.basename)
         # A couple extra in case parsing fails
-        max_games = self.max_size // self.samples_per_game + 10
+        max_games = (self.max_size // self.samples_per_game) + 480
         if len(games) > max_games:
             games = games[-max_games:]
```
Comment (brilee) [inline example_buffer.py]: you can do
`games = games[-max_games:]` without the conditional and it will just work.

File: example_buffer.py [hunk 2]
```diff
@@ -25,12 +25,12 @@
...
-MINIMUM_NEW_GAMES = 5000
+MINIMUM_NEW_GAMES = 12000
 
 
 def pick_examples_from_tfrecord(filename, samples_per_game=4):
     protos = list(tf.python_io.tf_record_iterator(filename, READ_OPTS))
-    if len(protos) < 20:  # Filter games with less than 20 moves
+    if len(protos) < 50:  # Filter games with less than 20 moves
         return []
     choices = random.sample(protos, min(len(protos), samples_per_game))
```
Comment (brilee) [inline example_buffer.py]: not going to go with the len(protos) // 50 idea?

File: example_buffer.py [hunk 3]
```diff
@@ -68,20 +68,18 @@ def parallel_fill(self, games, threads=8):
         """ games is a list of .tfrecord.zz game records. """
         games.sort(key=os.path.basename)
         # A couple extra in case parsing fails
-        max_games = self.max_size // self.samples_per_game + 10
+        max_games = (self.max_size // self.samples_per_game) + 480
```
Comment (brilee) [inline example_buffer.py]: why 480?

File: example_buffer.py (hunk 2 above)
Comment (amj) [reply example_buffer.py]: `// 50` is for when a game is in the window for ~50 generations, correct?  That's probably not the ratio anymore...

File: example_buffer.py (hunk 3 above)
Comment (amj) [reply example_buffer.py]: The consequences of not having enough positions was worse before #329

File: example_buffer.py (hunk 1 above)
Comment (amj) [reply example_buffer.py]: ack.

**Pull request merged**